node/src/components/tests_common.rs

```diff
@@ -0,0 +1,11 @@
+use std::time::Duration;
```

Comment:
We have a `testing` module in the root of node which is feature-gated behind `test` already.  I think this function should live there.